src/styles/ui/FormField.scss

```diff
@@ -0,0 +1,45 @@
+@import "../theme";
+
+.form-field {
+  display: flex;
+  flex-direction: column;
+  margin-bottom: 20px;
+
+  &.label {
+    font-size: 1.2em;
+    margin-bottom: 5px;
+  }
+
+  input {
+    padding: 10px;
+    font-size: 1.2em;
+    border: 1px solid #ccc;
+    border-radius: 5px;
+    background: $accent;
+    min-width: 15em;
+
+    &:focus {
+      outline: none;
+      border-color: #6c63ff;
+    }
+  }
+
+  textarea {
+    padding: 10px;
+    font-size: 1.2em;
+    border: 1px solid #ccc;
+    border-radius: 5px;
+    background: $accent;
+    min-height: 12em;
+
+    &:focus {
+      outline: none;
+      border-color: #6c63ff;
+    }
+  }
```

Comment:
![image](https://github.com/sopra-fs24-group07/sopra-fs24-group-07-client/assets/160509323/bb18b1c9-035a-4388-8c3a-d1454fb4c912)

Please set min/max height/width for textare

I also liked the old styling better, did you change it on purpose?

Reply:
1. Doesn't happen for me, could you share more details so I can recreate this issue?
2. Since I'm quite advanced with most of my issues/tasks I'm planning to go over the styling the upcoming days. I'd like to make everything a bit more modern, but would obviously revert it to the old styling if you prefers so.

Reply:
1. the is a draghandle in the bottom right corner of the textarea with which you can change its size. Either set max/min or deactivate that if possible
2. you can change the styling but at the moment it looks very unfinished and especially comments looks very off

Reply:
1. Done (also disabled some formfields when not in edit mode, was previously forgotten)
2. I understand the concern, I also get the part about the comments. for me it is important to set a visual boundary between the team information and the comments, would like to leave this as it is for now and do proper styling later. its easier to refactor the entire component now, before we're working on other stuff that might conflict again.